a3p-integration/proposals/n:upgrade-next/priceFeed-follower-auction.test.js

```diff
@@ -3,6 +3,7 @@ import { getDetailsMatchingVats } from './vatDetails.js';
 
 test('new auction vat', async t => {
   const details = await getDetailsMatchingVats('auctioneer');
-  // This query matches both the auction and its governor, so 2*2
-  t.is(Object.keys(details).length, 4);
+  // This query matches both the auction and its governor, so 2*3 for the
+  // original, the vaults&auctions coreEval, and the priceFeed coreEval.
+  t.is(Object.keys(details).length, 6);
```

Comment:
We talked about finding the old instances. It might be worth tracking the boardIDs in that vat spreadsheet.